Library/Formula/graphviz.rb

```diff
@@ -43,7 +43,7 @@ class Graphviz < Formula
 
   if build.with? "bindings"
     depends_on "swig" => :build
-    depends_on :python
+    depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
The URL seems to be down at the moment and I bashed my cache before I went offline last night, but there's a variable in `configure` which points to `config/config_python.py` in the tarball. Rather than letting it point there for that one call you should be able to `inreplace` it to:
```ruby
"#{lib}/python2.7/site-packages"
```